src/client-side-metrics/gcp-metrics-handler.ts

```diff
@@ -79,7 +91,7 @@ export class GCPMetricsHandler implements IMetricsHandler {
     // The projectId is needed per metrics handler because when the exporter is
     // used it provides the project id for the name of the time series exported.
     // ie. name: `projects/${....['monitored_resource.project_id']}`,
-    if (!this.otelInstruments) {
+    if (!this.projectToInstruments[projectId]) {
```

Comment:
nit: this might be a bit easier to read if you break out the instrument creation into a helper:

```
private createInstrumentsForProject(projectId: String): MetricInstruments {
    ...
}

private getInstruments(projectId: string): MetricsInstruments {
    if (!this.projectToInstruments[projectId]) {
       this.projectToInstruments[projectId] = createInstrumentsForProject(projectId)
    }
    return  this.projectToInstruments[projectId]
}
```

Reply:
Sure. Done.